packages/faceted-search/src/components/README.md

```diff
@@ -0,0 +1,324 @@
+
+| Props       | Type   | Info                                                |
+| ----------- | ------ | --------------------------------------------------- |
+| children    | func   | render props with current faceted mode as parameter |
+| error       | string | error returns by query call                         |
+| id          | string | id shared to all children                           |
+| facetedMode | string | mode display at first render                        |
+| inProgress  | bool   | indicates query progress                            |
+
+---
+
+### FacetedSearchIcon
+
+Simple **button icon** switch.
+
+| Props   | Type     | Info                            |
+| ------- | -------- | ------------------------------- |
+| active  | boolean  | icon is activated or not        |
+| loading | string   | display skeleton if true        |
+| onClick | function | Callback when button is clicked |
+
+---
+
+### FacetedToolbar
+
+The toolbar show the chosen **display mode** and the **switch buttons**.
+
+| Props               | Type   | Info                                                                          |
+| ------------------- | ------ | ----------------------------------------------------------------------------- |
+| children            | any    | children that will be render                                                  |
+| id                  | string | id                                                                            |
+| facetedMode         | string | You can choose between 'basic' and 'advanced'                                 |
+| onChangeFacetedMode | func   | Callback when user click on button. Return either 'basic' or 'advanced' value |
+| t                   | func   | i18n translate function                                                       |
+
+---
+
+## Badges
+
+Most badges will have props like this
+
+| Props                 | Type               | Info                                  |
+| --------------------- | ------------------ | ------------------------------------- |
+| id                    | string             | id                                    |
+| label                 | string             | label displayed on the left side      |
+| initialOpenedOperator | bool               | show / hide operator popover at mount |
+| initialOpenedValue    | bool               | show / hide value popover at mount    |
+| operator              | operatorPropTypes  | current operator                      |
+| operators             | operatorsPropTypes | list of operators                     |
+| size                  | string             | button icon size "small large"        |
+| value                 | string             | value displayed in the textarea       |
+| t                     | func               | i18n translate function               |
+
+### BadgeText
+
+A badge with a **text area**.
+
+| Default Operators |
+| ----------------- |
+| contains          |
+| endWidth          |
```

Comment:
not sure about this ones: `endWidth`, `notContains`, `regExp`, `startWidth`, where they comes from?